addons/microsoft_calendar/models/calendar.py

```diff
@@ -81,18 +99,25 @@ def _microsoft_to_odoo_values(self, microsoft_event, default_reminders=(), defau
         values = {
             **default_values,
             'name': microsoft_event.subject or _("(No title)"),
-            'description': microsoft_event.bodyPreview,
+            'description': plaintext2html(microsoft_event.bodyPreview),
```

Comment:
Do not use `plaintext2html` as in `14.0`, the `description` field is a `field.Text` and not yet a `field.Html` ;-)